fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/derby/CreateCommonTokenValuesTmp.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.derby;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.database.utils.api.IDatabaseStatement;
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+import com.ibm.fhir.schema.control.FhirSchemaConstants;
+
+
+/**
+ * Create the COMMON_TOKEN_VALUES_TMP table
+ */
+public class CreateCommonTokenValuesTmp implements IDatabaseStatement {
+    private static final Logger logger = Logger.getLogger(CreateCommonTokenValuesTmp.class.getName());
+    
+    @Override
+    public void run(IDatabaseTranslator translator, Connection c) {
+        
+        if (!isExists(c)) {
+            final String ddl = ""
+                    + "DECLARE GLOBAL TEMPORARY TABLE common_token_values_tmp ("
+                    + "  token_value VARCHAR(" + FhirSchemaConstants.MAX_TOKEN_VALUE_BYTES + "), "
```

Comment:
This is fixed to something like 255?

Reply:
Yeah, it matches the definition in the real table for this column.
    public static final int MAX_TOKEN_VALUE_BYTES = 1024;
which is the same length as the string column these fields used to be stored in.